pkg/logging/response_writer.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) The Thanos Authors.
```

Comment:
`responseWriter` could be part of HTTP util package, it seems like it is reusable.

Reply:
Done

Reply:
Done